client/src/pages/learn/responsive-web-design/index.md

```diff
@@ -4,7 +4,16 @@ superBlock: Responsive Web Design
 ---
 ## Introduction to Responsive Web Design
 
-This introduction is a stub
+Welcome to Responsive Web Design!
 
-Help us make it real on [GitHub](https://github.com/freeCodeCamp/freeCodeCamp/edit/master/client/src/pages/learn/responsive-web-design/index.md).
- 
+Let's first take a look at what is in the scope of Responsive Web Design (and what is not).
+
+The Web part of Responsive Web Design means that the content you are creating is designed to be delivered over the web, to people's browsers. To do this, you will need to learn the language that browsers use to describe web pages: HTML (Hypertext Markup Language) for content, and CSS (Cascading Style Sheets) for Designing.
+
+That brings us to design. Web Design focuses on how websites should look and behave to the end user. There are plenty of additional aspects to websites, including how to host a website so people can access it; how to protect your websites and store your users' data, and the ethos behind what should go on a website in the first place. These are somewhat harder to teach, but do require a good understanding of the capabilities of the web in modern contexts. 
+
+And what of the last word - Responsive? Well, that brings us to the idea of modern contexts. In today's world, websites are expected to be just as accessible on laptops, tablets and phones. If you compare your user experience on a document-editing program on a computer versus a document-editing program on a mobile, the features on mobile are likely to be far more constrained. This applies in some way to websites as well, but developers have found interesting ways to adapt to the mobile experience, not in the least by harnessing powerful features built for flexible (a.k.a responsive) web displays such as `css-flexbox` and `css-grid`.
```

Comment:
Do we really need to use the `css-` prefix with Flexbox and grid - 

> web displays such as `css-flexbox` and `css-grid`

It appears like they are literally called `css-flexbox` and `css-grid`. Maybe removing the `-` should fix it.